api/wasm-interpreter/src/lib.rs

```diff
@@ -194,6 +194,14 @@ impl WrappedInstance {
     pub fn hide(&self) {
         self.0.hide();
     }
+
+    /// THIS FUNCTION IS NOT PART THE PUBLIC API!
+    /// Highlights instances of the requested component
+    #[wasm_bindgen]
+    pub fn highlight(&self, path: &str, offset: u32) {
+        self.0.highlight(path.into(), offset);
```

Comment:
This broke the build of the vs code extension because of the unconditional call to a function behind the feature flag. I pushed https://github.com/slint-ui/slint/commit/587c9f694279c5ddf2acd587e03ca464446753bd as workaround but that might not be the correct fix - please check.